lib/mayaUsd/commands/baseListShadingModesCommand.cpp

```diff
@@ -59,6 +166,9 @@ MayaUSDListShadingModesCommand::createSyntax() {
     MSyntax syntax;
     syntax.addFlag("-ex", "-export", MSyntax::kNoArg);
     syntax.addFlag("-im", "-import", MSyntax::kNoArg);
+    syntax.addFlag("-eo", "-exportOptions", MSyntax::kString);
+    syntax.addFlag("-ea", "-exportAnnotation", MSyntax::kString);
+    syntax.addFlag("-fen", "-findExportName", MSyntax::kString);
```

Comment:
These new options help the UI build the material export dropdown menu and moves the knowledge about the nice name and annotation closer to the source.